backend/server/src/services/authService.ts

```diff
@@ -28,16 +28,16 @@ export class AuthService {
       .exec();
 
     if (user === null || user === undefined) {
-      throw new UnauthorizedError({
-        message: "Invalid email or password"
+      throw new BadRequestError({
+        message: "Invalid email"
       });
     }
 
     const isValidPassword = await user.checkPassword(password);
 
     if (!isValidPassword) {
-      throw new UnauthorizedError({
-        message: "Invalid email or password"
+      throw new BadRequestError({
+        message: "Invalid password"
```

Comment:
jos toi invalid email or password pilkotaan tolleen ni pitäskö olla mielummin no user found with that email?